index.bs

```diff
@@ -155,28 +156,38 @@ Parsing {#parsing}
 To <dfn abstract-op local-lt="parse header">obtain a response's embedder policy</dfn> given a
 [=response=] (|response|):
 
-1.  Let |policy| be "`unsafe-none`".
-
-2.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
+1.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
     |response|'s [=response/header list=].
 
-3.  If |header| is not `null`:
+2.  If |header| is `null`, then return a new [=/embedder policy=].
+
+3.  Set |header| to the result of [=isomorphic decoding=] |header|.
+
+4.  Let |parsed item| be the result of executing the [$Structured Header parsing algorithm$]
+    with <var ignore>input_string</var> set to |header|, and <var ignore>header_type</var> set
+    to "`item`".
+
+    If parsing fails, then return a new [=/embedder policy=].
+
+    ISSUE(httpwg/http-extensions#662): The ASCII requirements of Structured Headers are
+    somewhat unclear to me. Do we need to check whether |header| is an [=ASCII string=] before
+    passing it into the parsing algorithm?
+
+5.  Let |value| be the first component of |parsed item|.
 
-    1.  Set |header| to the result of [=isomorphic decoding=] |header|.
+6.  Let |parameters| be the second component of |parsed item|.
```

Comment:
`value` and `parameters` are `|parsed item|'s bare_item` and `|parsed item|'s parameters` respectively.

Reply:
Done.